util/query.py

```diff
@@ -339,10 +349,21 @@ def getNFTVolumes(
             break
         for order in new_orders:
             lastPriceValue = float(order["lastPriceValue"])
-            if lastPriceValue == 0:
-                continue
-            nft_addr = order["datatoken"]["nft"]["id"].lower()
             basetoken_addr = order["lastPriceToken"]
+            nft_addr = order["datatoken"]["nft"]["id"].lower()
+
+            if lastPriceValue == 0:
+                if len(order["dispensers"]) > 0:
```

Comment:
Rather than adding gas cost _only_ for free assets, I propose we add it for _all_ asset. That is, include priced assets too. Why:
- It better tracks the actual $ spent by the consumer to consume an asset. More realistic.
- It will simplify our code. E.g. here you don't need to test if "lastPriceValue == 0".